res/controllers/Numark-Scratch-scripts.js

```diff
@@ -0,0 +1,452 @@
+var NumarkScratch = {};
+
+/////////////////
+// Tweakables. //
+/////////////////
+
+// Beatloop Roll sizes for the 4 pads, for available values see:
+// https://manual.mixxx.org/2.3/en/chapters/appendix/mixxx_controls.html#control-[ChannelN]-beatloop_X_toggle
+NumarkScratch.autoLoopSizes = [
+    "0.25",
+    "0.5",
+    "1",
+    "2",
+];
+
+// Default is false. When set to false, the EncoderKnob manages looping. Shift + EncoderKnob scrolls the library/loads track
+// If set to true, the EncoderKnob scrolls the library/loads track. Shift + EncoderKnob manages looping.
+NumarkScratch.invertLoopEncoderFunction = false;
```

Comment:
This is really a necessary feature?

Reply:
With this encoder in Serato, the only option is to press shift + turn to scroll through the library and shift + push to load tracks, every damn time. The default is changing loop length and turning on loops, who uses loops that much, lol?
It's so unintuitive. and I rarely use loops (or haven't taken the time to learn). Numark should really have just added a browse encoder and load buttons. Rant over
Anyway, for this mapping inverting the encoder functions to non-standard is optional. I think this might be helpful to someone else.

Reply:
Thats a fair assesment. As long as it doesn't complicate the design too much, its fine.